### DISCUSSION TITLE
Free/dummy server link for example flutter app testing

### DISCUSSION THREAD

**Discussion by zzaman88** [ant-media/Ant-Media-Server#5209] [General]
Hi dear,
I want to test the example apps for Flutter
Is there any free or dummy server environment for testing the example Fullter application?

Thanks in advance!

**Comment by mekya** [comment]
Hi @zzaman88,
You can use test.antmedia.io for this kind of purposes. WebRTCAppEE is available there. 

https://test.antmedia.io/WebRTCAppEE/

FYI

Let me know if it's ok for you. 

Cheers

**Reply by zzaman88** [reply]
Thank you so much for your prompt response.
It's working.

But I am facing an issue to test the Peer sample app, the app is crashing when it is connected to the server.
Here is the log:

--------- beginning of crash
11:58:11.124  E  #
                 # Fatal error in: ../../pc/peer_connection.cc, line 823
                 # last system error: 0
                 # Check failed.
                 # 
11:58:11.125  A  Fatal signal 6 (SIGABRT), code -1 (SI_QUEUE) in tid 17728 (signaling_threa), pid 17285 (flutter_example)

[logs.txt](https://github.com/ant-media/Ant-Media-Server/files/11684601/logs.txt)

Could you please suggest to overcome this scenario?

**Reply by USAMAWIZARD** [reply]
Thanks @zzaman88 I am taking a look at this issue.

**Reply by USAMAWIZARD** [reply]
@zzaman88 please change the  ant_media_flutter version to  1.0.1 in pubspec.yaml file  previous version have some problem.

  ant_media_flutter: 1.0.1
